src/components/body/body-cell.component.ts

```diff
@@ -23,6 +25,46 @@ import { MouseEvent, KeyboardEvent } from '../../events';
           (click)="onCheckboxChange($event)"
         />
       </label>
+      <label class="clickable"
```

Comment:
Ideally this would be a button element for a11y.

Reply:
But button dominates the flexibility of design.

Reply:
I'm not sure I follow. You can make the button look anyway you want.

Reply:
Shall we provide an optional class for the button?